lib/node.js

```diff
@@ -468,7 +493,25 @@ declare module "punycode" {
 }
 
 declare module "querystring" {
-  // TODO
+  declare function stringify(
+    obj: Object,
+    separator?: string,
+    equal?: string,
+    options?: {
+      encodeURIComponent?: (str: string) => string;
+    }
+  ): string;
+  declare function parse(
+    str: string,
+    separator?: string,
+    equal?: string,
+    options?: {
+      decodeURIComponent?: (str: string) => string;
+      maxKeys?: number;
+    }
+  ): any;
+  declare function escape(str: string): string;
+  declare function unescape(str: string, decodeSpaces?: boolean): string;
```

Comment:
I'm not seeing the `decodeSpaces` param listed in the node API docs:

https://nodejs.org/api/querystring.html#querystring_querystring_unescape